site/content/blogs/release-v128.ja.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 'AWS Copilot v1.28: Static Site Service タイプの登場！'
+twitter_title: 'AWS Copilot v1.28'
+image: ''
+image_alt: ''
+image_width: '1051'
+image_height: '747'
+---
+
+# AWS Copilot v1.28: Static Site Service タイプの登場！
+
+投稿日: 2023 年 5 月 24 日
+
+AWS Copilot コアチームは Copilot v1.28 のリリースを発表します。
+本リリースにご協力いただいた [@interu](https://github.com/interu)、[@0xO0O0](https://github.com/0xO0O0)、[@andreas-bergstrom](https://github.com/andreas-bergstrom) に感謝します。
+私たちのパブリックな[コミュニティチャット](https://app.gitter.im/#/room/#aws_copilot-cli:gitter.im)は成長しており、オンラインでは 400 人以上、[GitHub](http://github.com/aws/copilot-cli/) では 2.9k 以上のスターを獲得しています。
+AWS Copilot へご支援、ご支持いただいている皆様お一人お一人に感謝をいたします。
+
+Copilot v1.28 では、いくつかの新機能と改良が施されています。
+
+- **Static Site Service タイプ**: AWS S3を使って静的な Web サイトをデプロイし、ホストすることができるようになりました。[詳細はこちらを確認してください。](#static-site-service-type).
+- **コンテナイメージの並列ビルド**: メインコンテナとサイドカーコンテナのそれぞれのイメージを並行してビルドできるようになりました。並列ビルドにより、ビルドと AWS ECR へのコンテナイメージのプッシュにかかる全体の時間を短縮することができます。
```

Comment:
"build main container and sidecar container images in parallel" should be translated into "~を並列にビルドできるようになりました" because [the term 並行 refers to cuncurrency](https://ja.wikipedia.org/wiki/%E4%B8%A6%E8%A1%8C%E6%80%A7) while [the term 並列 refers to parallel computing](https://ja.wikipedia.org/wiki/%E4%B8%A6%E5%88%97%E8%A8%88%E7%AE%97).